applications/nrf_desktop/src/events/ble_event.h

```diff
@@ -84,6 +97,8 @@ struct ble_discovery_complete_event {
 	struct event_header header;
 
 	struct bt_gatt_dm *dm;
+	u16_t pid;
+	u8_t peer_type;
```

Comment:
`enum peer_type peer_type`

Reply:
We do not assign enum values here. It is bitmask, enum indicates only bit positions in mask.